src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskCompleteByIdControllerTest.java

```diff
@@ -259,14 +259,13 @@ public void should_return_a_403_when_the_user_did_not_have_sufficient_permission
 
         result.then().assertThat()
             .statusCode(HttpStatus.FORBIDDEN.value())
-            .contentType(APPLICATION_JSON_VALUE)
-            .body("timestamp", lessThanOrEqualTo(ZonedDateTime.now().plusSeconds(60)
-                .format(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT))))
-            .body("error", equalTo(HttpStatus.FORBIDDEN.getReasonPhrase()))
-            .body("status", equalTo(HttpStatus.FORBIDDEN.value()))
-            .body("message", equalTo(
-                format("User did not have sufficient permissions to complete task with id: %s", taskId)
-            ));
+            .contentType(APPLICATION_PROBLEM_JSON_VALUE)
+            .body("type", equalTo(
+                "https://github.com/hmcts/wa-task-management-api/problem/role-assignment-verification-failure"))
+            .body("title", equalTo("Role Assignment Verification"))
+            .body("status", equalTo(403))
+            .body("detail", equalTo(
+                "Role Assignment Verification: Role assignment verifications failed."));
```

Comment:
detail value looks different to documentation? 
"detail": "The request failed the Role Assignment checks performed."